src/main/java/service/OperationService.java

```diff
@@ -0,0 +1,5 @@
+package service;
+
+public interface OperationService {
```

Comment:
```suggestion
public interface OperationHandler {
```
Use correct name for strategy class